website/api/generating-proofs/deterministic-builds.md

```diff
@@ -0,0 +1,35 @@
+# Deterministic Guest Program Builds
+
+In the context of RISC Zero application development, [Deterministic builds][term-deterministic-builds] are necessary to ensure a clear linkage between the source code for the [guest program][term-guest-program] and the resulting [Image ID][term-image-id].
+
+<!-- TODO: link to issue -->
+
+Deterministic builds are made possible by running the `rustc` compiler inside a Docker container.
+
+There are a few ways to access deterministic builds for your zkVM application:
+
+## Using `cargo risczero build`
+
+For CLI driven workflows, [`cargo risczero build`][cargo-risczero-rust-docs] has options to enable deterministic builds in scripts for CI and as one-off commands.
+
+## Using `build.rs`
```

Comment:
Before I approve, I'd like @SchmErik or @flaub to 👍 the details of the instructions lines 11 to 30 here.

Reply:
handing it off to you @pdg744 for what I hope is a PR ready to merge otherwise :crossed_fingers: 
I could not push to this branch anymore, so made a fork: https://github.com/risc0/risc0/pull/2459